src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -7,9 +7,13 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_ELDERLY_DISPLAYED_INDEX = "The elderly index provided is invalid";
+    public static final String MESSAGE_INVALID_VOLUNTEER_DISPLAYED_INDEX = "The volunteer index provided is invalid";
     public static final String MESSAGE_INVALID_NRIC = "The %s NRIC provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_NRIC_NOT_EXIST = "There is no such person with the given NRIC!";
+    public static final String MESSAGE_DUPLICATE_ELDERLY = "This elderly already exists in the database";
+    public static final String MESSAGE_DUPLICATE_VOLUNTEER = "This volunteer already exists in the database";
```

Comment:
In some other situations where you use string, such as `AddVolunteerCommand`, you use "New volunteer added: %s", where the %s will include all the details of the volunteer being added. Will this makes the message here too long and include too much details?

Reply:
I tried doing it but its not practical because you have to use ```String.format``` and the second argument you have to put ```"elderly"``` which is a magic string and to solve magic string you would have to declare ```String ELDERLY_STRING = "elderly"```. You have to do the same for volunteer and ultimately you end up with more strings